src/main/java/rs/hybridit/security/TokenHelper.java

```diff
@@ -0,0 +1,187 @@
+package rs.hybridit.security;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import rs.hybridit.model.User;
+
+@Component
+public class TokenHelper {
+
+	@Value("spring-security-demo")
```

Comment:
you might be using @Value incorrectly ... this is pretty useless 

https://www.baeldung.com/spring-value-annotation

Reply:
Changed it so that the field reference values defined in application.properties file.